graylog2-web-interface/src/views/components/searchbar/date-time-picker/TimeRangeAddToQuickListButton.tsx

```diff
@@ -0,0 +1,144 @@
+                 data-event-element="Add to quick list"
+                 data-testid="add-to-quick-list-popover">
+          <Input type="text"
+                 id="add-to-quick-list-description"
+                 placeholder="Add description..."
+                 title="Time range description"
+                 aria-label="Time range description"
+                 defaultValue={description}
+                 onChange={({ target: { value } }) => debounceHandleOnChangeDescription(value)}
+                 formGroupClassName="" />
+          {!!equalTimerange && (
+          <p>
+            <Icon name="exclamation-triangle" />
+            You already have similar time range in
+              {' '}
+            <Link to={Routes.SYSTEM.CONFIGURATIONS} target="_blank">Range configuration</Link>
+            <br />
+            <i>f.e. ({equalTimerange.description})</i>
+          </p>
+          )}
+          <ModalSubmit disabledSubmit={!description} submitButtonText="Add timerange" onCancel={toggleModal} onSubmit={onAddTimerange} bsSize="small" />
+        </Popover>
+      </Position>
+    </Portal>
+  );
+};
+
+const TimeRangeAddToQuickListButton = ({ timerange, isTimerangeValid }: {timerange: TimeRange, isTimerangeValid: boolean }) => {
+  const { userTimezone } = useUserDateTime();
+  const formTarget = useRef();
+  const { config, refresh } = useSearchConfiguration();
+  const [showForm, setShowForm] = useState(false);
+  const sendTelemetry = useSendTelemetry();
+
+  const toggleModal = useCallback(() => {
+    setShowForm((cur) => !cur);
+  }, []);
+  const addTimerange = useCallback((description) => {
+    const quickAccessTimerangePreset = { description, timerange: onSubmittingTimerange(timerange, userTimezone), id: generateId() };
+
+    ConfigurationsActions.update(ConfigurationType.SEARCHES_CLUSTER_CONFIG,
+      {
+        ...config,
+        quick_access_timerange_presets: [
+          ...config.quick_access_timerange_presets,
+          quickAccessTimerangePreset],
+      }).then(() => {
+      refresh();
+      toggleModal();
+    });
+
+    if (quickAccessTimerangePreset) {
+      sendTelemetry('form_submit', {
+        app_pathname: 'search',
+        app_section: 'search-bar',
+        app_action_value: 'add_to_quick_access_timerange_presets',
+        event_details: {
+          timerange: quickAccessTimerangePreset.timerange,
+          id: quickAccessTimerangePreset.id,
+        },
+      });
+    }
+  }, [config, refresh, sendTelemetry, timerange, toggleModal, userTimezone]);
+
+  const equalTimerange = useMemo(() => config
+    ?.quick_access_timerange_presets
+    ?.find((existingTimerange) => isTimerangeEqual(existingTimerange.timerange, onSubmittingTimerange(timerange, userTimezone))), [config, timerange, userTimezone]);
+
+  return (
+    <>
+      <Button disabled={!isTimerangeValid} title="Add timerange to quick access timerange list" ref={formTarget} onClick={toggleModal}>
+        <Icon name="plus" />
```

Comment:
Imo the save icon looks better here. We use the plus icon e.g. to add a search filter to the search and saving a quick access time range is not a similar process.